examples/gaussian_process/svgp.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+Sparse variational Gaussian process, loosely based on
+
+> Opper and Archambeau 2009, The variational Gaussian approximation revisited.
```

Comment:
I haven't read this but seems I can understand the most of the code. Where did the code use it?

Reply:
I wanted to emphasize the particular combination of model / inference I've implemented (Sparse GP with variational posterior, not e.g. the exact O(N^3) Gaussian process, or using the intractable exact posterior for inference). I'll update the comments to make it more clear.

Reply:
I see.